python/distrdf/dask/test_reducer_merge.py

```diff
@@ -252,6 +254,34 @@ def test_distributed_asnumpy_columns(self):
         int_32_dtype = numpy.dtype("int32")
         self.assertEqual(npy_x.dtype, int_32_dtype)
 
+    def test_distributed_asnumpy_lazy(self):
+        """Test that `AsNumpy` can be still called lazily in distributed mode"""
```

Comment:
Can we have a common helper function for `test_distributed_asnumpy` and `test_distributed_asnumpy` that accepts a `lazy` parameter? 90 % of the code of that helper function would be the same no matter the value of `lazy`.